catboost/libs/options/plain_options_helper.cpp

```diff
@@ -389,3 +389,92 @@ void NCatboostOptions::PlainJsonToOptions(
 
     trainOptions["flat_params"] = plainOptions;
 }
+
+void NCatboostOptions::OptionsToPlainJson(
```

Comment:
OptionsToPlainJson => ConvertOptionsToPlainJson - function name should start with a verb